src/pages/reader/epub.vue

```diff
@@ -368,13 +368,21 @@ export default {
           ),
         });
         const cfi = this.resumeFromLocalStorage();
-        this.rendition.themes.default({
-          body: {
-            '-webkit-text-size-adjust': 'none',
-            'text-size-adjust': 'none',
-            direction: 'ltr', // mitigate epubjs mixing up dir & page-progression-direction
-          },
-        });
+
+        const metadata = await this.book.loaded.metadata;
+        const bodyCSS = {
+          color: '#333',
```

Comment:
Why add #333 though

Reply:
liker.land default text color in bookstore
https://github.com/likecoin/liker-land/blob/23ff92e24d95647d42a719201aabc7926ebe026a/src/components/NFTBook/ItemCardV2.vue#L25